rust/prebuilt/transit_realtime.serde.rs

```diff
@@ -59,13 +59,13 @@ impl serde::Serialize for Alert {
             struct_ser.serialize_field("informedEntity", &self.informed_entity)?;
```

Comment:
There are a ton of changes here that I'm not sure the consequences of. I did not update any crates but perhaps they were updated earlier and not built?